diff --git a/node_modules/react-native-anchor-point/index.ts b/node_modules/react-native-anchor-point/index.ts
index f789fa7..3f7f236 100644
--- a/node_modules/react-native-anchor-point/index.ts
+++ b/node_modules/react-native-anchor-point/index.ts
@@ -59,3 +59,18 @@ export const withAnchorPoint = (transform: TransformsStyle, anchorPoint: Point,
 
     return { transform: injectedTransform };
 };
+
+export function getAnchorPoint(anchorPoint: Point, size: Size) {
+    let translateX = 0;
+    let translateY = 0;
+
+    if (anchorPoint.x !== defaultAnchorPoint.x && size.width) {
+        translateX = size.width * (anchorPoint.x - defaultAnchorPoint.x);
+    }
+
+    if (anchorPoint.y !== defaultAnchorPoint.y && size.height) {
+        translateY = size.height * (anchorPoint.y - defaultAnchorPoint.y);
+    }
+
+    return { translateX, translateY };
+}
